.azure/windows-stack.yml

```diff
@@ -5,10 +5,10 @@ jobs:
     vmImage: 'windows-2019'
   strategy:
     matrix:
-#     stack_810:
-#       STACK_YAML: "stack8101.yaml"
-#     stack_88:
-#       STACK_YAML: "stack88.yaml"
+      stack_810:
+       STACK_YAML: "stack8101.yaml"
```

Comment:
The name 8101 is very hard t parse. Could we have stack_8_10_1.yaml?

Reply:
Or `stack-8.10.1.yaml` like hls. But maybe we should change all stack.yaml's for consistency and if it is the case i would prefer to do it in another pr.

Reply:
Agreed - not really something for this PR